pom.xml

```diff
@@ -141,6 +141,10 @@
                         <groupId>org.jetbrains.kotlin</groupId>
                         <artifactId>kotlin-stdlib</artifactId>
                     </exclusion>
+                    <exclusion>
+                        <groupId>com.squareup.okio</groupId>
+                        <artifactId>okio</artifactId>
```

Comment:
in 6.2.x we use artifact `wire-schema` instead of `wire-schema-jvm` and we didn't support reserved fields for Protobuf.